src/views/Curators/Curator.vue

```diff
@@ -0,0 +1,534 @@
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Record maintainer(s)",
+                value: "recordMaintainers"
+              },
+              {
+                text: "Under curation by",
+                value: "curator"
+              }
+            ],
+            recordsWithoutDois: [
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Date created",
+                value: "createdAt"
+              },
+              {
+                text: "Date last edit",
+                value: "updatedAt"
+              },
+              {
+                text: "Created by",
+                value: "creator"
+              },
+              {
+                text: "Last edited by",
+                value: "lastEditor"
+              }
+            ],
+            hiddenRecords: [
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Date created",
+                value: "createdAt"
+              },
+              {
+                text: "Under curation by",
+                value: "curator"
+              },
+              {
+                text: "Created by",
+                value: "creator"
+              }
+            ]
+          },
+          search1: "",
+          search2: "",
+          search3: "",
+          search5: "",
+          search6: "",
+          search7: ""
+        }
+      },
+      computed: {
+        ...mapState('users', ['user', "messages"]),
+      },
+      created() {
+        this.$nextTick(function () {
+          this.recordType = recordTypes;
+        });
+      },
+      async mounted() {
+          await this.getUser();
+          if (this.messages()["getUser"].error){
+            this.setError({field:"login", message:"You've been logged out automatically"});
+            this.$router.push({path: "/accounts/login"})
+          }
+          client.setHeader(this.user().credentials.token);
+          let data = await client.executeQuery(getCurationRecords);
+          this.allDataCuration = data["curationSummary"];
+          client.initalizeHeader();
+          this.prepareData();
+      },
+      methods: {
+          ...mapActions('users', ['getUser', 'setError']),
+          prepareData(){
+            let _module = this;
+            _module.prepareApprovalRequired(_module.allDataCuration);
+            _module.prepareMaintenanceRequests(_module.allDataCuration);
+            _module.prepareRecordsInCuration(_module.allDataCuration);
+            _module.prepareRecordsWithoutDois(_module.allDataCuration);
+            _module.prepareHiddenRecords(_module.allDataCuration);
+            _module.prepareRecordsCuratorCreationsLastWeek(_module.allDataCuration);
```

Comment:
You don't really have to rename `this`. Assigning `this` to a new variable only makes sense if you use that variable inside an anonymous function if that function does not use arrow functions.
```suggestion
            this.prepareApprovalRequired(_module.allDataCuration);
            this.prepareMaintenanceRequests(_module.allDataCuration);
            this.prepareRecordsInCuration(_module.allDataCuration);
            this.prepareRecordsWithoutDois(_module.allDataCuration);
            this.prepareHiddenRecords(_module.allDataCuration);
            this.prepareRecordsCuratorCreationsLastWeek(_module.allDataCuration);
```